src/frontend/src/sass/_font.scss

```diff
@@ -1,6 +1,15 @@
-/* This section describes the default fonts we use in B.C gov
- */
-@font-face {
-  font-family: myFirstFont;
-  src: url('sansation_light.woff');
+.font__weight--700 {
+  font-weight: 700;
+}
+
+.text__align-center {
+  text-align: center;
+}
+
+.max-char-width {
+  max-width: 75ch;
+}
+
+.text__line-height--2 {
```

Comment:
let us assign this to the tag directly.

Reply:
Are `p` elements always `line-height: 2;`?

Reply:
they should be 1.5 I think. 

If the line height is different, can we play with margin or padding? any property related to font/text we should not micro configure. We can discuss with branding again.

Reply:
Currently, the default line-height works for body text. the line-height of two fits with the designs, where `p` is used for display text. The other way I can think of doing this is to create a class `p__display` and apply that instead of `text__line-height--2`. I was just trying to stay consistent with BEM conventions.